packages/components/src/font-size-picker/next/font-size-control.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * External dependencies
+ */
+import { contextConnect } from '@wp-g2/context';
+import { View, VisuallyHidden, VStack } from '@wp-g2/components';
+
+/**
+ * Internal dependencies
+ */
+import FontSizeControlSelect from './font-size-control-select';
+import FontSizeControlSlider from './font-size-control-slider';
+import { useFontSizeControl } from './use-font-size-control';
+
+function FontSizeControl( props, forwardedRef ) {
+	const {
+		customLabel = __( 'Custom' ),
+		disabled,
+		label,
+		max,
+		min,
+		options,
+		inputValue,
+		isDefaultValue,
+		value,
+		onChange,
+		onReset,
+		onInputChange,
+		selectDropdownProps,
+		size,
+		withSlider,
+		withNumberInput,
+		withSelect,
+		...otherProps
+	} = useFontSizeControl( props );
```

Comment:
Can we document the props of the component. Are these the exact same props as the existing component? What are the differences. For me personally, this is the most important thing when adding the G2 components:  make sure the props make sense, that we don't have useless props and unnecessary public APIs. Also, ideally the props don't have any DOM related thing to make them cross devices.

Reply:
Hi @youknowriad, the component was adding some customization props but I removed them. Now the components just accept the same props as the existing one. The easiest way to see the props that are accepted is by checking the hook associated with the component.
https://github.com/WordPress/gutenberg/blob/86b3188dad0e454e8400dab2f08e1133047249e6/packages%2Fcomponents%2Fsrc%2Ffont-size-picker%2Fnext%2Fuse-font-size-control.js#L28